main.markdown

```diff
@@ -365,21 +365,75 @@ Coefficiente binomiale
     * $0 \leq k \leq n \iff \binom{n}{k} = \frac{n!}{k!(n - k)!}$
 
 Combinazione
-:   Lorem ipsum
+:   | Il numero dei sottoinsiemi con $k$ elementi di un insieme di
+      $n$ elementi può essere espresso come:
+    | $C_{(n,k)} = \binom{n}{k} = \frac{n!}{k!(n - k)!}$ 
 
-Relazione di Stiefel
-:   Lorem ipsum
+Formula di Stiefel
+:   | La formula di Stiefel è definibile come la seguente uguaglianza:
+    | $\binom{n}{k} = \binom{n - 1}{k - 1} + \binom{n - 1}{k}$ con $1 \leq k \leq n$
+    | Caso particolare: $2^{n} = \sum\limits_{k=0}^{n}\binom{n}{k}$
+                
 
 Numeri di Bell
-:   Lorem ipsum
+:   | I numeri di Bell sono una serie di numeri definibili attraverso 
```

Comment:
Keep it simpe:

Definizione: Numeri di bell
Corpo: Sono una serie di numeri... (senza ripetere "i numeri di Bell")